azurelinuxagent/common/cgroup.py

```diff
@@ -38,39 +38,25 @@ class MetricsCounter(object):
     PROCESSOR_PERCENT_TIME = "% Processor Time"
     TOTAL_MEM_USAGE = "Total Memory Usage"
     MAX_MEM_USAGE = "Max Memory Usage"
+    THROTTLED_TIME = "Throttled Time"
 
 
 re_user_system_times = re.compile(r'user (\d+)\nsystem (\d+)\n')
 
 
-class CGroupContollers(object):
-    CPU = "cpu"
-    MEMORY = "memory"
-
-
 class CGroup(object):
-    @staticmethod
```

Comment:
This create method was weird: the caller needs to pass the type of the cgroup. Probably it was done to keep track of the controller (for use when formatting to string), but that would not be needed since the path includes it.